src/composables/offline/offline-layers.composable.ts

```diff
@@ -33,15 +42,22 @@ export default function useOfflineLayers() {
    * @see recreateOfflineLayer in v3
    */
   function createOfflineLayer(offlineLayerSpec: OfflineLayerSpec) {
-    if (
-      offlineLayerSpec.layerType === OfflineLayerTypeValue.LAYER_OFFLINE_TILE
-    ) {
-      const layer = initLayer(offlineLayerToLayer(offlineLayerSpec))
+    const validOfflineLayer = [
+      OfflineLayerTypeValue.LAYER_OFFLINE_TILE,
+      OfflineLayerTypeValue.LAYER_OFFLINE_BG_VECTOR,
+    ]
+
+    if (validOfflineLayer.indexOf(offlineLayerSpec.layerType) > -1) {
```

Comment:
includes ?

Reply:
yes thx for the suggestion, will update this in another PR